README.md

```diff
@@ -174,4 +174,3 @@ Features can be toggled by setting the relevant environment variable.
 | Name                              | Default Value | Type    | Description                                                                                                                                                        |
 |-----------------------------------|---------------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | WORK_AND_INTERESTS_TAB_ENABLED    | false         | Boolean | Set to true to enable the Work and Interests tab. Will be removed when feature is complete and released                                                            |
-| PRE_PRISON_QUALIFICATIONS_ENABLED | false         | Boolean | Set to true to enable the Pre-prison qualifications and education section on the Education and training tab. Will be removed when feature is complete and released |
```

Comment:
This should have been `OTHER_QUALIFICATIONS_ENABLED` after I renamed the variable. 🤦‍♂️